docker-compose.yml

```diff
@@ -35,3 +35,9 @@ services:
       KAFKA_TOOLS_LOG4J_LOGLEVEL: ERROR
     logging:
       driver: none
+  wiremock:
+    container_name: app_mock
+    ports:
+      - "8089:8089"
+    image: "wiremock/wiremock"
+    command: --port 8089
```

Comment:
after yesterday's commit i understand that this part isn't needed for unit testing. I have a conflict with ports.